feat: Attendance Status and some enhancements (#179)
frappe/erpnext_documentation: erpnext_documentation/www/docs/user/manual/en/human-resources/attendance.md

    <img class="screenshot" alt="Attendance" src="{{docs_base_url}}/assets/img/human-resources/marking_unmarked_attendance.gif">

> Introduced in Version 13

> ### 3.2 Remaining Half-day Status

Reviewer: ```suggestion
### 3.2 Remaining Half Day Status
```

> After Version 12, ERPNext started tracking the remaining half day status. On selecting Half Day [Attendance Status](/docs/user/manual/en/human-resources/attendance-status). Field **Remaining Half day Status** will appear, where you can select the Present or Absent Attendance Status for the remaining half day.

Reviewer: ```suggestion
In ERPNext version 13, you can track the remaining half-day status. On selecting Half Day [Attendance Status](/docs/user/manual/en/human-resources/attendance-status), a field called **Remaining Half Day Status** will be shown, where you can select the Attendance Status as "Present" or "Absent" for the remaining half day.
```

Author: Actually, there is a know hardcoded status for attendance status. need to discuss.

> <img class="screenshot" alt="Attendance" src="{{docs_base_url}}/assets/img/human-resources/remaining-half-day-status.png">

Reviewer: ```suggestion
<img class="screenshot" alt="Remaining Half Day Status" src="{{docs_base_url}}/assets/img/human-resources/remaining-half-day-status.png">
```

> > Note: If payroll is based on Attendance, Remaining Half Day Status will also contribute to your salary.

Reviewer: ```suggestion
> Note: If the Payroll is based on Attendance, the Remaining Half Day Status will also contribute to your salary.
```
